File: examples/test_GModelSpectralGauss.py
import gammalib

def test_a():
    model = gammalib.GModelSpectralGauss()
    energy = gammalib.GEnergy(1, "TeV")
    time = gammalib.GTime()
    print(model.eval(energy, time))
    
def test_mc():
    gammalib.GModelSpectralGauss.mc
    pass

def test_write():
    gammalib.GModelSpectralGauss.write
    pass

def test_flux():
    gammalib.GModelSpectralGauss.flux
    pass


if __name__ == '__main__':
    test_a()

File: examples/test_GModelSpectralGauss.cpp
#include <iostream>
#include "GammaLib.hpp"

void test_a() {
  GModelSpectralGauss model;
  GEnergy energy(1, "TeV");
  GTime time;
  std::cout << model.eval(energy, time) << std::endl;
}

void test_b() {
  GModelSpectralGauss model(42, GEnergy(3, "MeV"), GEnergy(1, "MeV"));
  GEnergy energy(3, "MeV");
  GTime time;
  std::cout <<  model.eval(energy, time) << std::endl;
  //std::cout <<  model.flux(GEnergy) << std::endl;
}


int main(void) {
  test_a();
  test_b();
}
